vm/ieleApi.ml

```diff
@@ -0,0 +1,157 @@
+open IeleClientUtils
+open Yojson.Basic.Util
+open Msg_types
+
+type block = {
+  state: (string * Yojson.Basic.json) list;
+  timestamp: string
+}
+
+let blocks = ref [{state=[];timestamp=""}]
```

Comment:
It's probably fine for now, but the asymptotic performance of the outer list is not good if we need indexing over blocks much, and the inner association list will be bad if there can ever be many accounts.

Reply:
I was aware of this issue but didn't view it as worth fixing because this code is intended to be for testing purposes only, and because most of the operations that actually occur in the solidity test harness interact only with the latest block, which is the head of the list and can be accessed in constant time.